src/core/models/dto/coreUnitDTO.ts

```diff
@@ -140,6 +140,11 @@ export interface ActivityFeedDto {
       code: string;
       shortCode: string;
     };
+    owner: {
+      code: string;
+      shortCode: string;
+      type: string;
+    };
```

Comment:
Move the params to a different type. Eg:

``` ts
interface CoreUnitParam  {
  coreUnit: {
    code: string;
    shortCode: string;
  }
}

interface OwnerParam {
  owner: {
    code: string;
    shortCode: string;
    type: string;
  }
}

type ActivityFeedParams = CoreUnitParam | OwnerParam
```

Reply:
You are right